### PR TITLE
chore: update effectively everything 

### PR DESCRIPTION
**Pull request by MimzyFang**
updated mods on 1.20.1
migrated 1.20.2 to 1.20.4 and updated mods 
1.20.2 is no longer supported
added a new lite version which is a clientside modpack that only includes a subset of mods and only supports them on the latest version it can

updated mc-modpack-kit to make building the pack possible again

### CODE REVIEW

**Comment by Sollace** [comment]
Rip the 5 players who still use 1.20.2

**Review by Sollace — APPROVED** [review]

**Comment by Sollace** [comment]
Don't forget to update the readme

**Comment by MimzyFang** [comment]
> Don't forget to update the readme

Will do Tomorrow AS I have a free day

File: README.md [hunk 1]
```diff
@@ -25,16 +25,18 @@ Compilation of all the different mods built to work with Mine Little Pony or tha
 
 ### By other devs
 1. ~[Farmer's Delight](https://modrinth.com/mod/farmers-delight-fabric)~ (only 1.20.1)
```
Comment (Sollace) [inline README.md]: Farmer's Delight Fabric is deprecated. With Unicopia updated you should be able to migrate to the new one with no problem.

https://modrinth.com/mod/farmers-delight-refabricated

File: README.md [hunk 2]
```diff
@@ -25,16 +25,18 @@ Compilation of all the different mods built to work with Mine Little Pony or tha
...
-7. [Continuity](https://modrinth.com/mod/continuity)
-8. [Enchantment Descriptions](https://modrinth.com/mod/enchantment-descriptions)
-9. [Zoomify](https://modrinth.com/mod/zoomify)
-10. ~[Sound Physics Remastered](https://modrinth.com/mod/sound-physics-remastered)~ (only 1.20.1)
-11. ~[Dynamic Lights](https://modrinth.com/mod/lambdynamiclights)~ (only 1.20.1)
+2. ~[Seasons Mod](https://modrinth.com/mod/fabric-seasons)~ (only 1.20.1)
+3. ~[Serene Seasons](https://modrinth.com/mod/serene-seasons)~ (only 1.20.4) 
+4. [Seasons Hud](https://modrinth.com/mod/seasonhud-fabric)
+5. [Falling Leaves](https://modrinth.com/mod/fallingleaves)
+6. ~[Missing Wilds](https://modrinth.com/mod/missing-wilds)~ (only 1.20.1)
```
Comment (Sollace) [inline README.md:32]: Missing Wilds has a 1.20.2 build available. Only in beta though

https://modrinth.com/mod/missing-wilds/version/v1.2.0-beta.7+1.20.1-quilt

Comment (MimzyFang) [reply README.md:32]: I dropped 1.20.2 as it is pretty much unused by most people and is not getting updates anymore from most mods

File: README.md (hunk 1 above)
Comment (MimzyFang) [reply README.md]: I actually already migrated just forgot to change the text give me a moment

**Comment by Sollace** [comment]
Alrighty, merge time! We good to go?

**Comment by MimzyFang** [comment]
> Alrighty, merge time! We good to go?

if you have nothing to complain about yeah

**Pull request merged**